### ISSUE TITLE
We can do better than that "logo" and tagline (font, css, ...)

### ISSUE THREAD

**Issue by benichu** [montrealrb/Montreal.rb#237]
![](http://storage.thouret.com.s3.amazonaws.com/monosnap/Events__Montreal.rb_2016-06-06_20-29-43.png)

**Comment by benichu** [comment]
@sophiedeziel any reason why we are not using that logo? 

![](https://raw.githubusercontent.com/montrealrb/Montreal.rb/master/app/assets/images/montreal-rb.png)

**Comment by sophiedeziel** [comment]
The red header + red part of the logo was pretty horrible and I decided to leave that for later. Anyone wanting to fix that is welcome!

**Comment by benichu** [comment]
should we try using that logo and experimenting with different header bgcolors?

**Comment by sophiedeziel** [comment]
Of course! I encourage anyone to experiment with the website, the style, the colors and see how they can improve it!

**Comment by benichu** [comment]
@sophiedeziel I started a pull request to experiment: https://github.com/montrealrb/Montreal.rb/pull/238

**Comment by KentCrutchfield** [comment]
The logo/colors look "ok" to me (maybe the font size for "Montreal's Ruby Community" could be bigger).

If we stick with this logo/color combo, the Twitter logo should change also...

**Comment by benichu** [comment]
personally never really liked that logo :) but I don't think we have an alternative for now...
will create a follow up ticket to update the social branding depending on the result of that visual refactor (if any)

**Comment by kloklo90** [comment]
![image](https://cloud.githubusercontent.com/assets/6946085/22048010/84302cac-dcf8-11e6-9c81-18db0ebe8bb1.png)
Would something like this suit the newer logo. I just put these together in 5 minutes on visio to be able to visualize wat it could look like.

**Comment by bellef** [comment]
How about something like that?
![selection_141](https://cloud.githubusercontent.com/assets/5313833/26061063/2cb64c90-3987-11e7-96b6-e69984f618f6.png)

**Comment by sophiedeziel** [comment]
I love it!

It should be aligned with the rest of the content, but I think that plain white version is what makes our logo look the best on a red background.

**Comment by bellef** [comment]
I'll make a PR about it when I got the logo right (contouring). ;)
